jupyter_sphinx/utils.py

```diff
@@ -74,12 +75,14 @@ def sphinx_abs_dir(env, *paths):
     # output_directory / jupyter_execute / path relative to source directory
     # Sphinx expects download links relative to source file or relative to
     # source dir and prepended with '/'. We use the latter option.
-    return "/" + os.path.relpath(
-        os.path.abspath(
-            os.path.join(output_directory(env), os.path.dirname(env.docname), *paths)
-        ),
-        os.path.abspath(env.app.srcdir),
-    )
+    out_path = (output_directory(env) /  Path(env.docname).parent / Path(*paths)).resolve()
+      
+    if os.name == "nt":
+        # Can't get relative path between drives on Windows
+        return out_path.as_posix()
```

Comment:
Does this actually produce a download link that works? I can't check on windows, but it seems that the filename has a different structure and I wonder if sphinx resolves this correctly.

Reply:
Pleasantly surprised that it works locally and when [hosted](https://foster999.github.io/test-jupyter-sphinx-win/)